geoapi/routes/__init__.py

```diff
@@ -7,18 +7,13 @@
 
```

Comment:
changes here are just related to just keeping our OpenAPI/swagger spec up-to-date (using `flask_restx`)